react/components/ItemContainer.js

```diff
@@ -28,16 +28,24 @@ export default class ItemContainer extends Component {
   }
 
   renderLinkFirstLevel(parentSlug, item) {
+    const { menuDisposition } = this.props
     const params = {
       department: parentSlug || item.slug,
     }
+
+    const firstLevelLinkClasses = classNames(`${categoryMenu.linkLevel2} db pv4 link no-underline outline-0 tl t-small truncate c-on-base underline-hover`, {

```

Comment:
`firstLevelLinkClasses` has `linkLevel2` classes?

Also, what do you think about changing all 2's to `second`'s? I find it more cool. ~naming variables with numbers remembers me of beginners in code`